### PR TITLE
Adds short notebook to check the grid features beyond 'tests'

### PR DESCRIPTION
**Pull request by MichaelClifford**
Most of the notebooks so far have been looking only at the data the makes up the actual grid visualizations. These can be found in the "tests" section of the json that constitutes the entire grid. There was some oversite on my part to check the consistency and availability of the other features across the grids. So, before digging much further into these potential additional features,  this is just a quick notebook that confirms they are all nearly identical across all grids for Red Hat as well as on the Kubernetes grids. 

Also impacts issue #15

### CODE REVIEW

**Review by aakankshaduggal — APPROVED** [review]
LGTM :+1:

File: notebooks/testgrid_feature_confirmation.ipynb [hunk 1]
```diff
@@ -0,0 +1,1871 @@
+{
```
Comment (SankBad) [inline notebooks/testgrid_feature_confirmation.ipynb:1]: <p>Hi Michael, </p><p> </p><p>Do we have a description of above features?</p><p>Can we add a link if it is available somewhere else?&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (MichaelClifford) [reply notebooks/testgrid_feature_confirmation.ipynb:1]: @SankBad I'm not sure if there is a complete description on these.  I think we will have to dig into them and see what is there to generate a description. The closest thing I could find was the [config docs](https://github.com/GoogleCloudPlatform/testgrid/blob/master/config.md) on the test grid repo.

Comment (SankBad) [reply notebooks/testgrid_feature_confirmation.ipynb:1]: got it :+1: , Thanks

**Pull request merged**